integrationtest/pom.xml

```diff
@@ -133,8 +133,6 @@
                 <dependency>
                     <groupId>javax.xml.bind</groupId>
                     <artifactId>jaxb-api</artifactId>
-                    <version>2.3.1</version>
-                    <scope>provided</scope>
```

Comment:
My intention was to utilize Maven's inheritance to reduce duplication and ensure the same version is used throughout the whole project. The same version and the same scope were repeated 4 times in different pom.xml files that all have the same parent pom.xml. So I thought it would be a good idea to move these things to the parent pom.xml.

Reply:
OK. No problem. I added the scope tags back to individual dependencies.